features/comments_and_kudos/guest_comments.feature

```diff
@@ -43,3 +43,26 @@ Scenario: Guest comments with embedded images are rendered as plain text
     And I should see "alt="
     And I should see "baz"
     But I should not see the image "src" text "https://example.com/image.jpg"
+
+Scenario: Guest sees warning footnote and required fields on comment form
+  Given the work "Test Work" by "author" with guest comments enabled
+  When I go to the work "Test Work"
+  Then I should see "You will not be able to edit or delete your comment after it is posted."
+  And I should see "Guest name (required)"
+  And I should see "Guest email (required)"
+  And I should see "All fields are required. Your name and comment text will both be publicly displayed. Your email address will not be made public, but it will be used to send you notifications of any replies to your comment."
+
+Scenario: Logged-in user does not see guest-specific elements
+  Given the work "Test Work" by "author" with guest comments enabled
+  And I am logged in as "commenter"
+  When I go to the work "Test Work"
+  Then I should not see "You will not be able to edit or delete your comment after it is posted."
+  And I should not see "Guest name (required)"
+  And I should not see "Guest email (required)"
+  And I should not see "All fields are required. Your name and comment text will both be publicly displayed. Your email address will not be made public, but it will be used to send you notifications of any replies to your comment."
+
+Scenario: Guest comment validation messages appear correctly
```

Comment:
This test is a bit out of place. The errors are created in the model, so this should be a model test. However, the changes here doesn't affect this behaviour, so I think you can just remove the test.

Reply:
Thanks Bilka2! This is my first time working with cucumber test and the comments are super helpful. I currently don’t have access to my computer, and will make revision in ~1 week when I do!